Tiltfile

```
@@ -115,6 +115,26 @@ COPY --from=tilt-helper /usr/bin/kubectl /usr/bin/kubectl
     },
 }
 
+# Create a data structure to hold information about addons.
+addons = {
+    "test-extension": {
+        "context": "./test/extension",
+        "image": "gcr.io/k8s-staging-cluster-api/test-extension",
+        "container_name": "extension",
+        "binary_name": "extender",
+        "namespace": "default",
+        "live_reload_deps": ["main.go", "handlers"],
+        "label": "RuntimeSDK",
+        "kustomize_substitutions": {
+            "TEST_EXTENSION_SERVICE_NAMESPACE": "default",
```

I think that using the "default name" will make things more generic/easier to use with different addons/extensions
```suggestion
            "SERVICE_NAMESPACE": "default",
```

---

I don't think we'll ever be able to make assumptions like that though - they'll always be pretty fragile - but I'm happy to change it back.